src/containers/CreateDoc/createDoc.css

```diff
@@ -0,0 +1,53 @@
+.page__content {
+  width: 100%;
+  max-width: 1024px;
+  margin: 0 auto;
+  display: grid;
+  grid-template: auto auto 1fr/ 30% 70%;
+  grid-gap: 15px;
+}
+
+/* LIST - Page content head */
+.page__content ul {
+  grid-column: 1 / 3;
+  display: flex;
+  justify-content: space-around;
+}
+.page__content ul li {
+  padding: 20px 30px;
+}
+
+/* INFORMATION - Left aside */
+.page__content__information {
```

Comment:
how's BEM  working here ? looks like `page` block could be avoided (or used in really general cases) over here you want `content` to be your block and `information` to be your element.

`content__information` is an option

I've seen other approaches like `layout` block, making this to be like 
`layout__information-bar` 
Keep in mind that you can use multi word with `-` in BEM

Reply:
BEM is always a mess in my head. So different ways to use it.
In this case, you are right, information is an element inside my content block. I mantained the "page" word to know the source of that content: Its the content of the page, as a global content. Otherwise, i thought content__information might be a little bit misleading (if read with no context at all - what information? Of what content?). Had i overthought it too much? please let me know and i change it :D :D

Reply:
The main need over here is not to add multiple blocks to the same name. So define your block (it could be page or content) and then your element (information-module or whatever you feel comfortable with. 

That's it : ) 

remember: `blocks-name__elements-name--modifier`